### PR TITLE
Comp setup dox

### PR DESCRIPTION
**Pull request by ydamit**
I think this covers some of the most important setup stuff!

### CODE REVIEW

File: parsons/airtable/airtable.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def get_records(self, fields=None, max_records=None, view=None, formula=None, so
         """
 
         # Raises an error if sort is None type. Thus, only adding if populated.
-        kwargs = {'fields': fields, 'max_records': max_records, 'formula': formula}
+        kwargs = {'fields': fields, 'max_records': max_records, 'view': view, 'formula': formula}
```
Comment (jburchard) [inline parsons/airtable/airtable.py]: What is this from?

File: useful_resources/computer_setup/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Computer Setup
+
+In this folder you will find instructions for setting up your computer in such a
```
Comment (jburchard) [inline useful_resources/computer_setup/README.md]: Perhaps include a link to the folder?

File: useful_resources/computer_setup/python3_setup.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+
+## Why Python 3?
+
+As you may have heard, starting [January 1, 2020][countdown] [Python 2.x is no
+longer officially supported][eol_article].
+
+Additionally, when using [Parsons][parsons], we use Python 3. Therefore, to make
+collaboration, debugging and maintenance easier, we encourage that you install and
+use Python 3 when writing scripts for TMC.
```
Comment (jburchard) [inline useful_resources/computer_setup/python3_setup.md]: Perhaps rephrase along the lines of: _Additionally, Parsons is only compatible with Python 3. So, in order to use parsons, you need to have Python 3 installed._

File: useful_resources/computer_setup/python3_setup.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+
+## Getting Started
+
+There are a number of ways to [install Python 3][install_py3]. Before you
+install it, check that what version you already have.
```
Comment (jburchard) [inline useful_resources/computer_setup/python3_setup.md]: _check what version, if any, you already have installed on your computer._

File: useful_resources/computer_setup/python3_setup.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+### Check Python Version
+
+1. Open the **Terminal** app
+2. Type `python --version`
+  * If you see something like `Python 3.x.x` then you are all set.
+    If not got to set 3
+3. Type `python3 --version`
+  * If you don't see any output then you need to install python 3.
```
Comment (jburchard) [inline useful_resources/computer_setup/python3_setup.md]: The rendering for the numbered list and then bullets looks a little funky. I'd consider trying something else.

File: useful_resources/computer_setup/python3_setup.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+There are a number of ways to [install Python 3][install_py3]. Before you
+install it, check that what version you already have.
+
+### Check Python Version
+
+1. Open the **Terminal** app
+2. Type `python --version`
+  * If you see something like `Python 3.x.x` then you are all set.
+    If not got to set 3
```
Comment (jburchard) [inline useful_resources/computer_setup/python3_setup.md]: If not, go to **step** 3.

File: useful_resources/computer_setup/python3_setup.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+3. Type `python3 --version`
+  * If you don't see any output then you need to install python 3.
+
+### Install Python with Homebrew
```
Comment (jburchard) [inline useful_resources/computer_setup/python3_setup.md]: This is only for Macs/Linux. You might want to specify.

File: useful_resources/computer_setup/terminal_app_setup.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+
+## Create A .bash_profile File
+
+### What is a .bash_profile file?
+
+This is often referred to as a _configuration script_. It can contain variable
+declarations, export variables, and commands to be executed when you start up
+your Terminal[*][ref1]. One common use case is store your API keys and tokens so
+they are not hard-coded in your scripts.
```
Comment (jburchard) [inline useful_resources/computer_setup/terminal_app_setup.md]: Which you should never do! All Parsons classes that require credentials allow you to set environmental variables, so you can just set these once and then forget about it.

File: useful_resources/computer_setup/terminal_app_setup.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+# Setting Up Your Terminal
```
Comment (jburchard) [inline useful_resources/computer_setup/terminal_app_setup.md]: I'd once again make it clear that this is Mac specific.

File: parsons/airtable/airtable.py (hunk 1 above)
Comment (ydamit) [reply parsons/airtable/airtable.py]: This was from another branch. Reverted it!

**Comment by elyse-weiss** [comment]
@ydamit should I review once you incorporate the suggestions from @jburchard ?

**Comment by ydamit** [comment]
@elyse-weiss I think I already incorporated Justin's comments a few days ago - but happy to do more if you see anything else that needs work!

**Pull request merged**